TCAT/Controllers/RouteDetail+DrawerViewController.swift

```diff
@@ -47,6 +47,12 @@ class RouteDetailDrawerViewController: UIViewController {
 
     var directionsAndVisibleStops: [RouteDetailItem] = []
     var expandedDirections: Set<Direction> = []
+    let notificationCellHeight: CGFloat = 70
+    let notificationTitles = [
+        Constants.Notification.notifyDelay,
+        Constants.Notification.notifyBeforeBoarding
+    ]
+    var sections: [RouteDetailSection] = []
```

Comment:
Let's name sections something more descriptive? "routeDetailSections" can work?

Reply:
imo might not be necessary. since the these sections are in the RouteDetailViewController already so i tihnk its safe to be implied